src/lms-harmonizer/tests_integration_pgsql/procs/harmonize_users/test_lmsusers_canvas.py

```diff
@@ -110,7 +110,7 @@ def it_should_run_successfully(test_db_config: PgsqlServerConfig):
                 "select edfistudentid, sourcesystemidentifier from lms.lmsuser",
             )
             assert len(LMSUser) == 2
-            assert LMSUser[0]["sourcesystemidentifier"] == SIS_ID
-            assert LMSUser[0]["edfistudentid"] == STUDENT_ID
-            assert LMSUser[1]["sourcesystemidentifier"] == NOT_MATCHING_SIS_ID
-            assert LMSUser[1]["edfistudentid"] is None
+            assert LMSUser[0]["sourcesystemidentifier"] == NOT_MATCHING_SIS_ID
```

Comment:
I wonder if this same sort order will come through on my system? Might need to add an `order by` clause on line 110. I'll let you know if everything passes on my system.